osu.Game.Rulesets.Tau/Objects/Slider.cs

```diff
@@ -100,15 +129,21 @@ protected override void CreateNestedHitObjects(CancellationToken cancellationTok
                         break;
 
                     case SliderEventType.Repeat:
-                        var time = (e.SpanIndex + 1) * SpanDuration;
-                        var node = Path.NodeAt((float)time);
-
                         AddNested(new SliderRepeat
                         {
                             ParentSlider = this,
                             RepeatIndex = e.SpanIndex,
-                            StartTime = StartTime + time,
-                            Angle = node.Angle
+                            StartTime = e.Time,
+                            Angle = angleAt((float)(e.Time - StartTime))
```

Comment:
What's wrong with using `Path.NodeAt()` here?

Reply:
The positioning messes up when the slider includes repeats

Reply:
I'm not sure what's wrong with changing the method inside that class instead of writing an entire new one...

Reply:
Because your Path implementation is kinda Scuffed and does not work for this use case senario.

Reply:
"Scuffed"? If it is so scuffed then I'd recommend you fix the actual method inside the path instead of doing this.

Reply:
"Scuffed"? If it is so scuffed then I'd recommend you fix the actual method inside the path instead of doing this.